x-pack/plugins/console_extensions/spec/overrides/data_frame.put_data_frame_transform.json

```diff
@@ -0,0 +1,25 @@
+{
+  "data_frame.put_data_frame_transform": {
+    "data_autocomplete_rules": {
+      "source": "SOURCE_INDEX_NAME",
```

Comment:
I'm not the most knowledgeable Console user, but do other APIs also pre-seed field names and values like this?

```
POST _data_frame/transforms/_preview
{
  "source": "SOURCE_INDEX_NAME",
  "pivot": {
    "aggregations": {
      "NAME": {}
    }
  }
}
```

Reply:
Yeah, this is pretty common,.